driver/bpf/maps.h

```diff
@@ -106,6 +106,13 @@ struct bpf_map_def __bpf_section("maps") rtt_static_map = {
         .max_entries = 65535,
 };
 
+struct bpf_map_def __bpf_section("maps") stash_tuple_map = {
+	.type = BPF_MAP_TYPE_PERCPU_HASH,
```

Comment:
I wonder if there exists the situation that the tuple is handled within different cpu, then the map of BPF_MAP_TYPE_PERCPU_HASH cannot share the tuple?